Apps/WebClient/src/ClientApp/app/router.ts

```diff
@@ -83,24 +84,13 @@ router.beforeEach(async (to, from, next) => {
       if (!hasAccess) {
         next({ path: "/login", query: { redirect: to.path } });
       } else {
-        let userIsRegistered: boolean = store.getters["auth/userIsRegistered"];
         // If the user is registerd and is attempting to go to registration re-route
+        let userIsRegistered: boolean = store.getters["user/userIsRegistered"];
         if (userIsRegistered && to.path === "/registration") {
```

Comment:
I think we need to include a check for the path /registrationInfo here too, its also part of the registration flow, if already registered go to timeline too

Reply:
added that path